src/Analyzer.JsonRuleEngine/JsonRuleEngine.cs

```diff
@@ -49,6 +49,57 @@ public static JsonRuleEngine Create(string rawRuleDefinitions, BuildILineNumberR
             return new JsonRuleEngine(ParseRuleDefinitions(rawRuleDefinitions), jsonLineNumberResolverBuilder);
         }
 
+        /// <summary>
+        /// Modifies the rules to run based on values defined in the configurations file.
+        /// </summary>
+        /// <param name="configuration">The template configurations to filter and analyze.</param>
+        public void FilterRules(string configuration)
+        {
+            var filteredRules = new List<RuleDefinition>();
+
+            ConfigurationDefinition contents;
+            try
+            {
+                contents = JsonConvert.DeserializeObject<ConfigurationDefinition>(configuration);
+            }
+            catch (Exception e)
+            {
+                throw new JsonRuleEngineException($"Failed to parse configurations file.", e);
+            }
+
+            if (contents.InclusionsConfigurationDefinition != null)
+            {
+                var includeSeverities = contents.InclusionsConfigurationDefinition.Severity;
+                var includeIds = contents.InclusionsConfigurationDefinition.Ids;
+                foreach (RuleDefinition rule in RuleDefinitions)
+                {
+                    if (includeSeverities.Contains(rule.Severity) || includeIds.Contains(rule.Id))
```

Comment:
Just want to verify the logic here.  If the provided config file has:
```
"inclusions": {
    "severity": [ 1 ]
    "ids": [ "SomeRuleId" ]
}
```
... the result will be running all rules with severity 1 *and* 'SomeRuleId', even if that rule has a different severity than 1.  Is this what we want?  Should we make this behavior explicit in documentation?

Reply:
Yes, that is what we want. I believe I have done that in this latest update.